server/models/flashcard.js

```diff
@@ -1,13 +1,21 @@
 import mongoose from "mongoose";
```

Comment:
for mocha/chai test, we will need to use `require` instead of `import`. inconveniently this applies to all files :( more details on [my #backend msg on testing](https://discord.com/channels/981018785806975016/987113366495559720/988695423084527618)